vbr/vbrCore/params/Params_Electric.m

```diff
@@ -0,0 +1,222 @@
+function [params] = Params_Eletric(method,GlobalParams)
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+  %
+  % params = Params_Electric(method)
+  %
+  % loads the parameters for an elastic method
+  %
+  % Parameters:
+  % ----------
+  % method    the method to load parameters for. If set to '', will return
+  %           limited information
+  % GlobalParams   option GlobalParams flag, not required and not currently used
+  %
+  % Output:
+  % ------
+  % params    the parameter structure for the electric method
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+  params.possible_methods={'yosh2009','SEO3','poe2010','sun2019','wang2006','UHO2014','DK2014'};
+
+  if ~exist('GlobalParams')
+    GlobalParams = Params_Global();
+  end
+
+  if strcmp(method,'yosh2009')
+    params.func_name='yosh2009'; % the name of the matlab function
+    % Yoshino et al, 2009
+    % Ionic Conduction
+    params.yosh2009_ol.S_i = 10^4.73; % S/m
```

Comment:
These `params` need a slight adjustment:

So when `Params_Electric` gets called, it puts this main `params` structure as the field like this:

```
VBR.in.electric.yosh2009 = Params_Electric('yosh2009');
```

That means when you add another structure on `params` like you're doing here, the output structure will be accessed like

```
VBR.in.electric.yosh2009.yosh2009_ol.S_i
... etc ...
```

So you should either update all your functions to reflect this nested structure **or** just change these lines to 
```suggestion
    params.S_i = 10^4.73; % S/m
```
I have a slight preference to the second option if there's not a reason to add the nested structures.